components/raftstore-v2/src/operation/command/admin/split.rs

```diff
@@ -171,6 +171,9 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
     pub fn update_split_flow_control(&mut self, metrics: &ApplyMetrics) {
         let control = self.split_flow_control_mut();
         control.size_diff_hint += metrics.size_diff_hint;
+        if self.is_leader() {
+            self.add_pending_tick(PeerTick::SplitRegionCheck);
```

Comment:
Now that we are using this more often, what happens if there're duplicate pending ticks?

Reply:
Nothing happen, it's by design to filter out duplicated ticks.